doc/ci_admin_guide/List_of_VMs_in_Windows_CI.md

```diff
@@ -137,3 +150,5 @@
     * TODO: Backup procedure?
     * TODO: DR procedure for critical VMs?
     * TODO: hardware monitoring?
+
+[docker-registry-provisioning]: Docker_registry_provisioning.md
```

Comment:
Maybe move this link under bullet point about `winci-registry`?

Reply:
Note that these links are not displayed in rendered documentation, the convention is to put them at the bottom of file / chapter.